### PR TITLE
normalize_uv generalization 

### PR DESCRIPTION
**Pull request by jeffjennings**
Generalizes `utilities/normalize_uv` to handle pointwise wavelengths

### CODE REVIEW

File: frank/utilities.py [hunk 1]
```diff
@@ -347,6 +348,11 @@ def normalize_uv(u, v, wle):
     logging.info('  Normalizing u and v coordinates by provided'
                  ' observing wavelength of {} m'.format(wle))
 
+    if type(wle) is not float or int:
+        if len(wle) != len(u):
+            raise ValueError("len(wle) = {}. It should be"
+                             "equal to len(u) = {}".format(len(wle), len(u)))
+
```
Comment (rbooth200) [inline frank/utilities.py:355]:
```suggestion
    wle = np.array(wle, dtype='f'8)
    if  len(wle) != 1 and len(wle) != len(u):
        raise ValueError("len(wle) = {}. It should be equal to "
                         "len(u) = {} (or 1 if all wavelengths are the same)".format(len(wle), len(u)))
```
This version handles lists properly, and numpy will do the type checking.

**Review by rbooth200 — CHANGES_REQUESTED** [review]

File: frank/utilities.py (hunk 1 above)
Comment (jeffjennings) [reply frank/utilities.py:355]: Perfect, thank you.

File: frank/utilities.py [hunk 2]
```diff
@@ -347,6 +348,10 @@ def normalize_uv(u, v, wle):
     logging.info('  Normalizing u and v coordinates by provided'
                  ' observing wavelength of {} m'.format(wle))
 
+    wle = np.array(wle, dtype='f8')
```
Comment (rbooth200) [inline frank/utilities.py:351]:
```suggestion
    wle = np.atleast_1d(wle, dtype='f8')
```

**Review by rbooth200 — COMMENTED** [review]
I checked, array should be atleast_1d..

**Review by rbooth200 — APPROVED** [review]

**Pull request merged**